applications/portal/backend/api/repositories/search_repository.py

```diff
@@ -135,7 +135,7 @@ def fts_and_filter_search(page: int = 0, size: int = 10, search: str = '', filte
 
     if antibodies_count < MAX_SORTED:
         # if sorting is not specified, we sort by the order of the ids
-        if search:
+        if order_by_string(filters):
```

Comment:
It is updated. 

`if search:` would allow only if the search text is present, and would not allow sorting if filters are present, but search doesn't. 

`if order_by_string(filters):` - does exactly as the comment above mentions - which is - "if sorting is not specified, we sort by the order of the ids"

Reply:
fts_sorting requires search, as search adds the ranking. Without search, it's a different logic

Reply:
created `apply_plain_sorting` to apply sorting when no fts search exists.